beets/plugins.py

```diff
@@ -129,14 +129,9 @@ def _set_log_level_and_params(self, base_log_level, func):
         value after the function returns). Also determines which params may not
         be sent for backwards-compatibility.
         """
-        if six.PY2:
-            argspec = inspect.getargspec(func)
-            func_args = argspec.args
-            has_varkw = argspec.keywords is not None
-        else:
-            argspec = inspect.getfullargspec(func)
-            func_args = argspec.args
-            has_varkw = argspec.varkw is not None
+        argspec = inspect.getfullargspec(func)
+        func_args = argspec.args
+        has_varkw = argspec.varkw is not None
```

Comment:
I think this auxiliary variables should be removed, and the right-hand side of these assignments substituted where they're used (which happens only once in each case).